### ISSUE TITLE
Search Results Hidden By The "Show All Excerpts Button" Are Reachable By Keyboards

### ISSUE THREAD

**Issue by Santoz2** [ManifoldScholar/manifold#2755]
Manifold's Reader Interface shortens search results by hiding excerpts in sections under a "SHOW ALL EXCERPTS" button. While they are visibly hidden, they can be reached by keyboards, making the process confusing for visual keyboard users, as it appears that focus is lost. This also makes the process lengthy for non-visual users, as results can not be effectively hidden.

## Expected Behavior
If content can be expanded and collapsed, the content should be hidden and unreachable when collapsed. 

Though not a perfect match to this issue, the below example of an accordion can be referred to for expected behaviors; any content that is collapsed should not be reachable by a keyboard.
[https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html)

## Current Behavior
When search results are collapsed (hidden behind the "SHOW ALL EXCERPTS" button), keyboard users can still reach the hidden results.

![image](https://user-images.githubusercontent.com/43553368/83057455-c47f8f80-a00b-11ea-8378-b764c0f4a31d.png)

## Steps to Reproduce
1. Navigate to the Reader Interface
2. Navigate to a project with readable and searchable text
3. Navigate into a text contained in that project
4. Navigate to and activate the search button in the menu
5. Enter any search criteria into the edit field and submit the search
6. Navigate to a search result before a "SHOW ALL EXCERPTS" button
7. With a keyboard, press Tab. Notice how focus does not move to the "SHOW ALL EXCERPTS" button. A visual keyboard user would not realize that focus went to a hidden search result. Conversely, a non-visual keyboard user is given a lengthy experience because they cannot collapse search results.

## Testing Environment
NVDA 2020.1
ESR Firefox 68.8.0 esr

**Comment by zdavis** [comment]
Resolved in #3082
